### DISCUSSION TITLE
Azure DevOps build and a .Net6 solution issue

### DISCUSSION THREAD

**Discussion by dpgindler** [wixtoolset/issues#6993] [Questions]
I am trying to figure out why our solution builds fine locally on a laptop using VS2022 with .NET6 SDK.
but when I push the changes into an azure devops pipeline using the VM, windows-latest, the code project builds fine, but the Wix Installer build fails due to not being able to find some of the .netcore runtime.   Like the one here:   Microsoft.AspNetCore.App.Runtime.win-x64

This package says it should not really be included as a package reference...  
Per this:   https://www.nuget.org/packages/Microsoft.AspNetCore.App.Runtime.win-x64#readme-body-tab

The error occurs during msbuild.  Here is the error from the build.

2022-11-09T21:50:30.0106799Z   dotnet publish "..\..\xyz\xyz.csproj" -c Release -o ..\..\publish --self-contained true -r win-x64
2022-11-09T21:50:30.4369070Z   MSBuild version 17.3.2+561848881 for .NET
2022-11-09T21:50:31.0592450Z     Determining projects to restore...
2022-11-09T21:50:32.2812840Z     Retrying 'FindPackagesByIdAsync' for source 'https://pkgs.dev.azure.com/xyz/_packaging/c125374b-3b22-4f84-85c7-802b236a3a3e/nuget/v3/flat2/microsoft.netcore.app.runtime.win-x64/index.json'.
2022-11-09T21:50:32.4353001Z     Response status code does not indicate success: 401 (Unauthorized).

By the way, this whole solution built fine before it was adjusted to use .NET6.

I am wondering if this is a wix issue ( seems unlikely since it builds fine on a laptop ), or if this is an azure devOps issue.. or I need to adjust my yaml file for something....   

Appreciate the input.

**Comment by robmen** [comment]
`Response status code does not indicate success: 401 (Unauthorized).` suggests you have an authentication problem from your CI server to your Azure DevOps NuGet feed. WiX isn't involved in any part of that.

**Reply by dpgindler** [reply]
Note this solution built fine in before we switched to .NET6.    

It seems a 401 is also returned when the file can not be found in package source it is trying to get it from.
We made several changes to our yaml file and removed a dotnet restore, and added a Nuget Authenticate

We know that dotnet restore was fine for the solution build, but when used for the wix part, it seemed to have an issue.

After this and a few changes we think did not make a difference, the installer built without errors.
